packages/asc-ui/src/components/ErrorMessage/ErrorMessage.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import ErrorMessage from './ErrorMessage'
+
+export default {
+  title: 'Experimental/Atoms/ErrorMessage',
+  decorators: [
+    (storyFn: () => React.ReactNode) => (
+      <div
+        style={{
+          display: 'flex',
+          maxWidth: 400,
+          flexDirection: 'column',
+          alignItems: 'flex-start',
+          padding: '40px 10px',
+        }}
+      >
+        {storyFn()}
+      </div>
+    ),
+  ],
+}
+
+export const Default = () => <ErrorMessage message="Dit is dus fout" />
```

Comment:
Nitpicking: this package is using only English text throughout the app

Reply:
gedaan